deploy-tool/sample.config.toml

```diff
@@ -0,0 +1,15 @@
+# DNS provider settings
+[dns]
+provider = "cloudflare" # options: cloudflare,
+
+    [dns.cloudflare]
```

Comment:
Is the indentation significant?

Reply:
No, it's generally up to us if we want the indentation for sub keys. I've generally seen it without indentation so I'll remove them.